packages/block-editor/src/components/inner-blocks/button-block-appender.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import BaseButtonBlockAppender from '../button-block-appender';
+import withClientId from './with-client-id';
+
+export const ButtonBlockAppender = function( { clientId } ) {
```

Comment:
Noting given you've got some mixed use: Typically I'd never expect to see a function keyword outside a function declaration in ES2015+ (i.e. [function expression](https://developer.mozilla.org/en-US/docs/web/JavaScript/Reference/Operators/function)). Either `function ButtonBlockAppender() {` or `const ButtonBlockAppender = () => {`.

As an arrow function, you could also take advantage of implicit return for compactness' sake:

```jsx
export const ButtonBlockAppender = ( { clientId } ) => (
	<BaseButtonBlockAppender rootClientId={ clientId } />
);
```

Reply:
I've changed these to be consistent.